### ISSUE TITLE
Pidgin icon has black background

### ISSUE THREAD

**Issue by rombert** [davidedmundson/xembed-sni-proxy#8]
I have pidgin installed and also the the pidgin-indicator extension which provides an appindicator icon for piding. Now that I've instealld xembed-sni-proxy, both icons show up. I noticed though that the xembed-sni-proxy icon looks slightly different and also has a blackground.

![pidgin-icons-differences](https://cloud.githubusercontent.com/assets/21237/10537566/502bd90e-73fb-11e5-8edf-afee851cd73b.png)

**Comment by FadeMind** [comment]
@rombert Dropbox have this issue too.

**Comment by rombert** [comment]
@FadeMind , yup, that's in my screenshot as well, but I though the Pidgin one was more telling. Dropbox is also undersized.

And to be complete, I'm running xembed-sni-proxy-0.0.1git~20151001~0d6ed03-1.1.x86_64 on openSUSE Tumbleweed.

**Comment by sherpya** [comment]
Dropbox icon background is ok, but it looks like upscaled
![indicator](https://cloud.githubusercontent.com/assets/120074/10559870/1d204450-74fa-11e5-920f-2e8b4808abf2.png)

**Comment by cuihaoleo** [comment]
Sometimes Hexchat icon's background turns black, too.

Latest Plasma 5.4.2 on Arch Linux, and latest git code.

**Comment by Scimmia22** [comment]
Claws Mail as well

**Comment by Crazy-Hopper** [comment]
Tried running Claws Mail with my fork. Looks OK. Couldn't test Pidgin though.
